problems/test/problem1_test.rb

```diff
@@ -0,0 +1,26 @@
+require "minitest/autorun"
+require_relative "../lib/problem1"
+
+class Problem1Test < MiniTest::Unit::TestCase
+  def setup
+    @problem1 = Problem1.new

```

Comment:
I used the approach to create and use a module level method instead, for this problem.